specification/servicenetworking/resource-manager/Microsoft.ServiceNetworking/preview/2024-05-01-preview/TrafficController.json

```diff
@@ -0,0 +1,1895 @@
+          "$ref": "#/definitions/FrontendProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "FrontendListResult": {
+      "type": "object",
+      "description": "The response of a Frontend list operation.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The Frontend items on this page",
+          "items": {
+            "$ref": "#/definitions/Frontend"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "FrontendProperties": {
+      "type": "object",
+      "description": "Frontend Properties.",
+      "properties": {
+        "fqdn": {
+          "type": "string",
+          "description": "The Fully Qualified Domain Name of the DNS record associated to a Traffic Controller frontend.",
+          "readOnly": true
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "Provisioning State of Traffic Controller Frontend Resource",
+          "readOnly": true
+        }
+      }
+    },
+    "FrontendUpdate": {
+      "type": "object",
+      "description": "The type used for update operations of the Frontend.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "Resource tags.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "enum": [
+        "Provisioning",
+        "Updating",
+        "Deleting",
+        "Accepted",
+        "Succeeded",
+        "Failed",
+        "Canceled"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Provisioning",
+            "value": "Provisioning"
+          },
+          {
+            "name": "Updating",
+            "value": "Updating"
+          },
+          {
+            "name": "Deleting",
+            "value": "Deleting"
+          },
+          {
+            "name": "Accepted",
+            "value": "Accepted"
+          },
+          {
+            "name": "Succeeded",
+            "value": "Succeeded"
+          },
+          {
+            "name": "Failed",
+            "value": "Failed"
+          },
+          {
+            "name": "Canceled",
+            "value": "Canceled"
+          }
+        ]
+      }
+    },
+    "ResourceId": {
```

Comment:
>ResourceId [](http://example.com/codeflow?start=5&length=10)

If this is a valid arm id, "format": "arm-id" can be added ?

Reply:
sure, added

Reply:
Removing this suggestion since its a breaking change